### PR TITLE
Update Internationalization Docs

### PR DESCRIPTION
**Pull request by thkim1011**
See title.

### CODE REVIEW

**Comment by sfshaza2** [comment]
Closing this for now. (Freezing the website.)

File: src/development/accessibility-and-localization/internationalization.md [hunk 1]
```diff
@@ -264,9 +325,199 @@ return MaterialApp(
...
+  "placeholders": {
+    "date": {
+      "type": "DateTime",
+      "format": "yMd"
+    }
+  }
+}
+```
+
+In an app, when the locale is US English, the following expression would produce  “7/10/1996”. If the locale was ‘Russian’, then it would produce “10.07.1996”.
```
Comment (Piinks) [inline src/development/accessibility-and-localization/internationalization.md:516]: If Russian is wrapped in ` `` `, should US English be too?

File: src/development/accessibility-and-localization/internationalization.md [hunk 2]
```diff
@@ -416,6 +667,29 @@ MaterialApp(
 );
 ```
 
+### Configuring the l10n.yaml file
+The `l10n.yaml` file allows you to configure the `gen-l10n` tool, such as where all the input files are located,
```
Comment (Piinks) [inline src/development/accessibility-and-localization/internationalization.md:671]: Newline

**Review by Piinks — COMMENTED** [review]
I know this is closed during the website freeze, but here is some feedback for when it is reopened.

**Comment by Piinks** [comment]
@thkim1011 the website is not frozen anymore. Do you want to reopen this PR?

**Pull request closed without merging**